data.js

```diff
@@ -208,6 +208,53 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their Guidelines \n- Contracts signed \n- literature wrote \n- Press Mentions",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of (2018)[https://www.wri.org/annualreport/2018-19#page=4] and their are (more)[https://www.wri.org/our-work/outcomes].",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        activeCountries: {
+            content: "Offices (global)[https://www.wri.org/annualreport/2018-19] - Headquarter is in Washington, DC.",
```

Comment:
"They have [offices globally][https://www.wri.org/annualreport/2018-19], their headquarter is in Washington, DC."

Reply:
done but with the right comibnation of () & []